integreat_cms/cms/templates/contacts/contact_list.html

```diff
@@ -74,7 +74,7 @@ <h1 class="heading">
                     {% empty %}
                         <tr>
                             <td colspan="8" class="px-4 py-3">
-                                {% translate "No contacts available yet." %}
+                                {% translate "No contacts archived yet." %}
```

Comment:
```suggestion
                                {% if is_archive %}
                                    {% translate "No contacts archived yet." %}
                                {% else %}
                                    {% translate "No contacts available yet." %}
                                {% endif %}
```

Now we have the opposite problem :smile:

Reply:
Oops, you're right!